scripts/rm_synthesis.py

```diff
@@ -29,32 +35,35 @@ def rmfit_quad(archive, phase_min, phase_max):
     phase_max: float
         The maximum phase to use to fit, should be a float between 0 and 1
     """
-    commands=["rmfit"]
+    commands = ["/pawsey/mwa/singularity/dspsr/dspsr.sif"]
```

Comment:
This looks wrong, shouldn't there be a singularity in here? Also it will likely be best to handle singularity containers the same way that we have with the beamformer. Put the container location in the vcstools/config.py and if there is a container then wrap the command with the singulairty command.

Reply:
Nope, it doesn't work if I do the 'singularity exec' method. This way works just fine. I've added the psrchive container to config.py